crates/symbolicator/src/utils/gcs.rs

```diff
@@ -1,10 +1,12 @@
 //! Access to Google Cloud Storeage
 
+use anyhow::Result;
```

Comment:
You are using `map_err` to explicitly convert to `GcsError`. Might as well continue using that instead of also introducing a new `DownloadError::Other` variant.

Reply:
oh right.  that was quite the brain failure, thanks for spotting